README.md

```diff
@@ -44,7 +44,7 @@ Please view the source for more information.
 
 Make sure that when used, myFT styles are included in the head stylesheet of the app. If not, necessary styles can be included by adding this code:
 ```
-@import 'n-myft-ui/myft-common/main';
+@import '@financial-times/n-myft-ui/myft-common/main';
```

Comment:
@juancancela `n-myft-ui/myft-common` doesn't have any stylesheet though and this package's `main.scss` includes `@financial-times/n-myft-ui/myft/main`...
https://github.com/Financial-Times/n-myft-ui/tree/main/myft-common

Reply:
Indeed. Updating doc.

Reply:
Maybe this should be point `@import '@financial-times/n-myft-ui/components/follow-button/main';` because the demo needs to add the style?

Reply:
However I wonder why the style is not included in this package?